spec/models/message_draft_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe MessageDraft do
+  context 'with valid attributes' do
+    subject { described_class.new(params) }
+
+    let(:params) { attributes_for :message_draft_for_model }
+    let(:other) { described_class.new(attributes_for(:message_draft_for_model)) }
+
+    it 'populates attributes' do
+      expect(described_class.attribute_set.map(&:name)).to contain_exactly(:id, :category, :subject, :body,
+                                                                           :attachment, :sent_date,
+                                                                           :sender_id, :sender_name, :recipient_id,
+                                                                           :recipient_name, :read_receipt)
+      expect(subject.id).to eq(params[:id])
+      expect(subject.category).to eq(params[:category])
+      expect(subject.subject).to eq(params[:subject])
+      expect(subject.body).to eq(params[:body])
+      expect(subject.attachment).to eq(params[:attachment])
+      expect(subject.sent_date).to eq(Time.parse(params[:sent_date]).utc)
+      expect(subject.sender_id).to eq(params[:sender_id])
+      expect(subject.sender_name).to eq(params[:sender_name])
+      expect(subject.recipient_id).to eq(params[:recipient_id])
+      expect(subject.recipient_name).to eq(params[:recipient_name])
+      expect(subject.read_receipt).to eq(params[:read_receipt])
```

Comment:
Since these are just inherited from Message model, I'm not sure we need to test them again. And if we do test them, I think we should look at using shared examples for the two.